### DISCUSSION TITLE
Integrating Adadmin with Angular

### DISCUSSION THREAD

**Discussion by thrasos** [giuliopons/adadmin#47] [Q&A]
![image](https://user-images.githubusercontent.com/2338139/199954853-f1671aae-f31a-4434-babf-a434a9e67765.png)

We are trying to integrate with Angular, but we are getting an error response; hence the code is not rendered.

Any tips on this?

**Comment by giuliopons** [comment]
What do you mean with "Integrating with Angular"? 
AdAdmin is made with PHP and creates some "Position" scripts that show banners.

This is a script position:

`<div id="AADIV31"></div><script>/* Square box fixed */ var k=decodeURIComponent(document.cookie),ca=k.split(';'),psc="";for(var i=0;i<ca.length;i++){var c=ca[i];while(c.charAt(0)==' ') c=c.substring(1);if (c.indexOf("adcapban")==0)psc+=(psc==""?"":",")+c.replace("adcapban","").replace("=",",");}var s = document.createElement("script");s.src = "https://www.barattalo.it/amb/ser.php?t=AADIV31"+String.fromCharCode(38)+"f=31"+String.fromCharCode(38)+"psc=" + psc;document.head.appendChild(s);</script>`

You should put this script in the HTML page where you want to show banners.
The ser.php file ouputs a javascript software that run in browser and is not build to run server side with Angular.

Hope this helps.
